### DISCUSSION TITLE
How can I copy the exact prompt back to txt2img prompt?

### DISCUSSION THREAD

**Discussion by teherauto** [AIrjen/OneButtonPrompt#41] [Q&A] [answered]
I love this plugin, but sometimes I can't copy the right text back to the txt2img prompt, because often uses the [ | ] choice list and I don't see which tag use in the generation.
What is the simplest method copy the right prompt back to the txt2img prompt?
Thanks! :-)

**Comment by AIrjen** [comment, marked as answer]
Hi, you could copy the prompt below the image grid. 
You could also drag the image into the prompt field, this will fill it with all the generation info.
